node-notes-cli/update.js

```diff
@@ -0,0 +1,28 @@
+import { readFileSync } from 'node:fs';
+import { readFile, writeFile } from 'node:fs/promises';
+export default update;
+
+const data = readFileSync('data.json', 'utf-8');
+const dataParse = JSON.parse(data);
+
+const number = process.argv[3];
+const message = process.argv[4];
+
+function update() {
+  readFile('data.json')
+    .then(() => {
+      for (const property in dataParse.notes) {
```

Comment:
Same on this one. This would work without the loop with just `dataParse.notes[number] = message;`.